deploy/04_operator.yaml

```diff
@@ -29,6 +29,10 @@ spec:
       containers:
         - name: custom-domains-operator
           image: REPLACE_ME/custom-domains-operator:latest
+          resources:
+            limits:
+              cpu: "100m"
+              memory: "1Gi"
```

Comment:
Can you share some screenshots in the PR of usage that lead to picking these values?

Reply:
Done

Reply:
Based on the screenshot, shouldn't the values be closer to something like 

```
cpu: "10m"
memory: "100Mi"
```

Reply:
After further discussion, these are upper limits for the operator, and don't have to be _super_ tightly configured.